Xamarin.Forms.Platform.iOS/CollectionView/ObservableItemsSource.cs

```diff
@@ -228,103 +258,32 @@ void Move(NotifyCollectionChangedEventArgs args)
 				var oldPath = NSIndexPath.Create(_section, args.OldStartingIndex);
 				var newPath = NSIndexPath.Create(_section, args.NewStartingIndex);
 
+				// update the internal items source
+				var item = _internalItemsSource[args.OldStartingIndex];
+				_internalItemsSource.Remove(item);
+				_internalItemsSource.Insert(args.NewStartingIndex, item);
+
+				// update the collection view
 				_collectionView.MoveItem(oldPath, newPath);
 				return;
 			}
 
 			var start = Math.Min(args.OldStartingIndex, args.NewStartingIndex);
-			var end = Math.Max(args.OldStartingIndex, args.NewStartingIndex) + count;
+			var end = Math.Max(args.OldStartingIndex + args.OldItems.Count, args.NewStartingIndex + count);
```

Comment:
move did not work with multiple items. I don't know which implementations of INCC support this, but now it works